src/pages/editor/flow/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Col, Row } from 'antd';
+import GGEditor, { Flow } from 'gg-editor';
+
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import React from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+import EditorMinimap from './components/EditorMinimap';
+import { FlowContextMenu } from './components/EditorContextMenu';
+import { FlowDetailPanel } from './components/EditorDetailPanel';
+import { FlowItemPanel } from './components/EditorItemPanel';
+import { FlowToolbar } from './components/EditorToolbar';
+import styles from './index.less';
+
+GGEditor.setTrackable(false);
+
+export default () => (
+  <PageHeaderWrapper
+    content={formatMessage({
+      id: 'editorandflow.description',
+      defaultMessage: 'description',
+    })}
+  >
+    <GGEditor className={styles.editor}>
+      <Row type="flex" className={styles.editorHd}>
+        <Col span={24}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 24](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)